fk/luse/src/test/java/se/inera/intyg/common/luse/v1/model/converter/certificate/question/QuestionFunktionsnedsattningAnnanTest.java

```diff
@@ -0,0 +1,176 @@
+import static se.inera.intyg.common.luse.v1.model.converter.RespConstants.FUNKTIONSNEDSATTNING_ANNAN_SVAR_ID;
+import static se.inera.intyg.common.luse.v1.model.converter.RespConstants.FUNKTIONSNEDSATTNING_ANNAN_SVAR_JSON_ID;
+import static se.inera.intyg.common.luse.v1.model.converter.RespConstants.FUNKTIONSNEDSATTNING_ANNAN_TEXT_ID;
+import static se.inera.intyg.common.luse.v1.model.converter.RespConstants.FUNKTIONSNEDSATTNING_CATEGORY_ID;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import se.inera.intyg.common.services.texts.CertificateTextProvider;
+import se.inera.intyg.common.support.facade.model.CertificateDataElement;
+import se.inera.intyg.common.support.facade.testsetup.model.CommonElementTest;
+import se.inera.intyg.common.support.facade.testsetup.model.config.ConfigTextAreaTest;
+import se.inera.intyg.common.support.facade.testsetup.model.validation.ValidationTextTest;
+import se.inera.intyg.common.support.facade.testsetup.model.value.ValueTextTest;
+
+@ExtendWith(MockitoExtension.class)
+class QuestionFunktionsnedsattningAnnanTest {
+
+    @Mock
+    private CertificateTextProvider texts;
+
+    @BeforeEach
+    void setup() {
+        when(texts.get(any(String.class))).thenReturn("Test string");
+    }
+
+    @Nested
+    class IncludeCommonElementTests extends CommonElementTest {
+
+        @Override
+        protected CertificateDataElement getElement() {
+            return QuestionFunktionsnedsattningAnnan.toCertificate(null, 0, texts);
+        }
+
+        @Override
+        protected String getId() {
+            return FUNKTIONSNEDSATTNING_ANNAN_SVAR_ID;
+        }
+
+        @Override
+        protected String getParent() {
+            return FUNKTIONSNEDSATTNING_CATEGORY_ID;
+        }
+
+        @Override
+        protected int getIndex() {
+            return 0;
+        }
+    }
+
+    @Nested
+    class IncludeConfigTextAreaTests extends ConfigTextAreaTest {
+
+        @Override
+        protected CertificateTextProvider getTextProviderMock() {
+            return texts;
+        }
+
+        @Override
+        protected CertificateDataElement getElement() {
+            return QuestionFunktionsnedsattningAnnan.toCertificate(null, 0, texts);
+        }
+
+        @Override
+        protected String getTextId() {
+            return FUNKTIONSNEDSATTNING_ANNAN_TEXT_ID;
+        }
+
+        @Override
+        protected String getDescriptionId() {
+            return FUNKTIONSNEDSATTNING_ANNAN_DESCRIPTION_ID;
+        }
+
+        @Override
+        protected String getJsonId() {
+            return FUNKTIONSNEDSATTNING_ANNAN_SVAR_JSON_ID;
+        }
+    }
+
+    @Nested
+    class IncludeAccordionTests {
```

Comment:
Should we extract this to a specific testFixture. Would be nice to reuse the test in a simple way without copy/paste.

This comment is of course related to all places where we have Accordion :-)